app/helpers/measures_helper.rb

```diff
@@ -1,4 +1,6 @@
 module MeasuresHelper
+  CHEG_SERVICE_URL = 'https://www.check-duties-customs-exporting-goods.service.gov.uk'.freeze
```

Comment:
I think we're not supposed to use the CHIEG acronym anymore.

```suggestion
  CHECK_DUTIES_SERVICE_URL = 'https://www.check-duties-customs-exporting-goods.service.gov.uk'.freeze
```

Reply:
https://www.urbandictionary.com/define.php?term=Chieg